sale_order_customer_order_number/models/sale_order_line.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 Quartile Limited
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import api, fields, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    customer_order_number = fields.Char("Customer Order Number")
```

Comment:
```suggestion
    client_order_ref = fields.Char("Customer Order Number")
```
If anything, I prefer to use the same name as the field in sale.order. But it's not a strong preference.